packages/runtime/test/lib/testUtils.ts

```diff
@@ -52,7 +54,7 @@ import { TestRuntimeServices } from "./RuntimeServiceProvider";
 import { TestNotificationItem } from "./TestNotificationItem";
 
 export async function syncUntil(transportServices: TransportServices, until: (syncResult: SyncEverythingResponse) => boolean): Promise<SyncEverythingResponse> {
-    const finalSyncResult: SyncEverythingResponse = { messages: [], relationships: [] };
+    const finalSyncResult: any = {};
```

Comment:
avoid any

Reply:
I made the method generic so that in future it dose not need to be changed when we add new properties to SyncEverythingResponse. The any is used because otherwise there will be TS errors.

Reply:
yeah - please still avoid any!

Reply:
Ok but than we need to go back to the old implementation.
Or use `// @ts-expect-error`

Reply:
fo me the old one was more readable.